local-cli/generator-windows/index.js

```diff
@@ -149,7 +149,7 @@ module.exports = yeoman.Base.extend({
   },
 
   end: function () {
-    const projectPath = path.resolve(this.destinationRoot(), 'windows', this.name);
+    //const projectPath = path.resolve(this.destinationRoot(), 'windows', this.name);
```

Comment:
There's a bunch of other warnings like that but I didn't want to remove code without someone else double checking.